wagtail/tests/test_signals.py

```diff
@@ -97,3 +98,49 @@ def test_signal_not_emitted_on_page_move(self):
 
         # Check the signal was NOT fired
         self.assertEqual(handler.call_count, 0)
+
+
+class TestCopyForTranslationDoneSignal(WagtailTestUtils, TestCase):
+    """
+    Tests for the `wagtail.signals.page_slug_changed` signal
```

Comment:
```suggestion
    Tests for the `wagtail.signals.copy_for_translation_done` signal
```

(Will fix this when I merge...)